index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/b2291adafa.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile version</title>
+</head>
+<body>
+ <section class="main-container">
+    <Section class="toolbar">
+        <Nav>
+            <h3>My Logo</h3>
+            <span class="material-icons">menu</span>
+        </Nav>
+    </Section>
```

Comment:
- [ ]  Your Toolbar container section should be `header`. Just replace 'section' with 'header'. This will make your code look very well structured.